tests/tck/features/bugfix/RewriteEdgeFilter.feature

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2022 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License.
+Feature: Test match used in pipe
+
+  Background:
+    Given a graph with space named "nba"
+
+  Scenario: Order by after match
+    When executing query:
+      """
+      match (v)-[e:like|teammate{start_year: 2010}]->() where id(v) == 'Tim Duncan' return e
```

Comment:
In openCypher, this edge filter means to find these edges which contain like or teammate labels and have the property start_year equal to 2010. So I think this filter should be converted to following format in NebulaGraph:

```
("like" in tags(e) and e.like.start_year = 2010) or ("teammate" in tags(e) and e.teammate.start_year=2010)
```

Reply:
This is not only `*.prop = xxxx`, but we should have an equivalent representation of `*.prop` of edge property consists multiple edge types.